valheim-backup

```diff
@@ -50,14 +50,27 @@ backup() {
         debug "No Valheim worlds to backup"
         return
     fi
-    backup_file="$BACKUPS_DIRECTORY/worlds-$(date +%Y%m%d-%H%M%S).zip"
-    pre_backup_hook "$backup_file"
-    info "Backing up Valheim server worlds to $backup_file"
+
     mkdir -p "$BACKUPS_DIRECTORY"
     chmod "$BACKUPS_DIRECTORY_PERMISSIONS" "$BACKUPS_DIRECTORY"
-    zip -r "$backup_file" "worlds/"
-    chmod "$BACKUPS_FILE_PERMISSIONS" "$backup_file"
-    post_backup_hook "$backup_file"
+
+    # Wenn BACKUPS_ZIP true, dann zip
+    if [ ${BACKUPS_ZIP} = true ]; then
+        backup_file="$BACKUPS_DIRECTORY/worlds-$(date +%Y%m%d-%H%M%S).zip"
+        pre_backup_hook "$backup_file"
+        info "Backing up Valheim server worlds to $backup_file"
+        zip -r "$backup_file" "worlds/"
+        chmod "$BACKUPS_FILE_PERMISSIONS" "$backup_file"
+        post_backup_hook "$backup_file"
+    else
+        find worlds/ -name "${WORLD_NAME}.db" -type d | fatal "Could not find world file for world ${WORLD_NAME}"
```

Comment:
This should likely be an OR `||` not a pipe and also not `fatal`. Fatal will end the backup script. Meaning on a fresh install with no existing worlds the backup script would just error out instead of trying again later.